services/mgmt/adhybridhealthservice/src/package_2014_01/operations.rs

```diff
@@ -166,7 +166,7 @@ pub mod adds_services {
     use super::{models, API_VERSION};
     pub async fn list(
         operation_config: &crate::OperationConfig,
-        filter: Option<&str>,
+        u24filter: Option<&str>,
```

Comment:
Where did the u24 come from?

Reply:
`$filter` becomes `u24filter`. The `u24` represents the `$`. http://www.unicode-symbol.com/u/0024.html

Reply:
Details are in https://github.com/Azure/azure-sdk-for-rust/blob/8ea32f9193fb468686b869247fd4aa0bc799197a/services/autorust/codegen/src/identifier.rs#L65-L85

Reply:
Thanks for the clarification on how this is generated.

That is not obvious from the perspective of users of the library.  Probably just as important though, it's not how other SDKs solve this.  

Ref:

1. Azure SDK For Go:
https://github.com/Azure/azure-sdk-for-go/blob/b63dc6cf8623c3e970b8e2ee5a16f948f3a96636/services/batch/2020-09-01.12.0/batch/jobschedule.go#L866-L879
2. Azure SDK for Python:
https://github.com/Azure/azure-sdk-for-python/blob/596e2de8cd39d1de34e0715ef1f8c47c93c4bf31/sdk/batch/azure-batch/azure/batch/operations/_job_schedule_operations.py#L1007-L1010
3. Azure SDK for Net: https://github.com/Azure/azure-sdk-for-net/blob/4162f6fa2445b2127468b9cfd080f01c9da88eba/sdk/batch/Microsoft.Azure.Batch/src/GeneratedProtocol/Models/JobListPreparationAndReleaseTaskStatusOptions.cs#L37-L54

Reply:
Thanks for the links. I agree. I made the necessary adjustments and added more Identifier unit tests. It is tricky.